README.md

```diff
@@ -737,6 +737,10 @@ Default is undef (array). Array of webbapp hosts. eg.: ['10.0.2.2:80', '10.0.2.3
 
 Default is undef (array). Array of carbonlink hosts. eg.: ['10.0.2.2:80', '10.0.2.3:80']
 
+##### `gr_carbonlink_hashing_type`
+
+Default is 'carbon_ch' (string). Define 'consistent-hashing' type.
```

Comment:
Default is 'undef'. Defines consistent-hashing type for 0.9.16+ eg.: 'carbon_ch'

Reply:
Should be good to go now! :)